packages/components/.stylelintrc.js

```diff
@@ -0,0 +1,150 @@
+    'scss/at-mixin-argumentless-call-parentheses': 'always',
+
+    // no strong opinions, probably OK to conform to suggested format
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/at-rule-conditional-no-parentheses/README.md
+    'scss/at-rule-conditional-no-parentheses': null,
+
+    // not sure why we would want this rule active as is by default, probably needs some tweaking (so my suggestion is to simply disable it)
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/double-slash-comment-empty-line-before/README.md
+    'scss/double-slash-comment-empty-line-before': null,
+
+    // while in theory this is a good idea, probably I would turn off this rule because it's buggy and triggers an error for us that doesn't make sense
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/operator-no-unspaced/README.md
+    // (many issues https://github.com/stylelint-scss/stylelint-scss/issues?q=is%3Aissue+is%3Aopen+%22operator-no-unspaced%22)
+    'scss/operator-no-unspaced': null,
+
+    // probably a good idea, but we have the make sure this will not impact consumers (may be using old versions of Sass?)
+    // see: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/no-global-function-names/README.md
+    'scss/no-global-function-names': null,
+
+    // using the default one doesn't work for us (we're using BEM)
+    // found this pattern here: https://github.com/humanmade/coding-standards/pull/199
+    'selector-class-pattern': ['^(?<block>(?:[a-z][a-z0-9]*)(?:-[a-z0-9]+)*)(?<element>(?:__[a-z][a-z0-9]*(?:-[a-z0-9]+)*))?(?<modifier>(?:--[a-z][a-z0-9]*)(?:-[a-z0-9]+)*)?$', { "resolveNestedSelectors": true }],
+
+    // the 'always-multi-line' option works for me
+    // see: https://stylelint.io/user-guide/rules/list/selector-list-comma-newline-after/
+    'selector-list-comma-newline-after': 'always-multi-line',
+
+    // strong-ish option here to disable this check: I prefer being explicit vs implicit (but if there are stronger opinions, I can live with it)
+    // see: https://stylelint.io/user-guide/rules/list/shorthand-property-no-redundant-values/
+    'shorthand-property-no-redundant-values': null,
+
+    // I would stick to the one we're using ("double") which is also the "standar" one (https://github.com/stylelint/stylelint-config-standard/blob/c12b8cb2b9a57e8dbd2696dd7ac2f78e72b829a4/index.js#L162)
+    // see: https://stylelint.io/user-guide/rules/list/string-quotes/
+    // 'string-quotes': 'double',
+    'string-quotes': null,
+
+    // since there's an issue with "currentColor" (https://github.com/stylelint/stylelint/issues/5863) I propose to simply ignore it
+    // see: https://stylelint.io/user-guide/rules/list/value-keyword-case/
+    'value-keyword-case': ['lower', {'ignoreKeywords': ['currentColor']}],
+
+    // =============================
+    // CONTROVERSIAL/HARD-TO-DECIDE:
```

Comment:
@alex-ju @KristinLBradley @MelSumner @Dhaulagiri this block of rules are the ones for which I am not sure what to do (probably I would simply disable them for now) but before deciding I want to hear your feedback

Reply:
These are the general rules I follow for formatting:

Always indent and separate by blank lines:
```
.a,
.b {
  color: blue;
  font-weight: bold;
}

.c {
  color: red;
  font-style: italic;
}
```

Use single line or indent as seems clearer. Separate by blank line or not as seems clearer (case by case.):
```
.a {color: red;}
.b {color: blue;}

.c {font-weight: bold;}

.d {
	font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
}
```

Reply:
I have no preference for formatting, as long as the linter does it for me and I don't have to hand-edit everything. 👍